.containers/mysql/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.1'
+
+services:
+
+  db:
```

Comment:
@mbentz-uf can you add phpmyadmin and default schema to this? i can't test the code by running it using the steps provided in the pull request.

Reply:
Done!